src/pages/zoeken/[slug]/[ie]/index.tsx

```diff
@@ -723,14 +737,116 @@ const ObjectDetailPage: NextPage<ObjectDetailPageProps> = ({ title, url }) => {
 		);
 	};
 
+	const renderMaintainerMetaTitle = ({
+		maintainerName,
+		maintainerLogo,
+		maintainerSlug,
+	}: IeObject): ReactNode => {
+		return (
+			<div className="p-object-detail__metadata-maintainer-title">
+				<p className="p-object-detail__metadata-name">
+					{tText('modules/ie-objects/const/index___aanbieder')}
+				</p>
+				<p className="p-object-detail__metadata-pill">
+					<TagList
+						className="u-pt-12"
+						tags={mapKeywordsToTags([maintainerName])}
+						onTagClicked={(keyword: string | number) => {
+							router.push(
+								stringifyUrl({
+									url: `/${ROUTE_PARTS.search}`,
+									query: {
+										[VisitorSpaceFilterId.Maintainer]: maintainerSlug,
```

Comment:
Yes correct, my bad. 

I'll add it but comment it out because https://github.com/viaacode/hetarchief-client/pull/720 should be merged before that code will work :)